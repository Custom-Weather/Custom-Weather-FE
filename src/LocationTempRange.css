#high-temp-img{
  position: fixed;
  height: 5%;
  width: 3%;
  top: 25%;
  left: 43%;
  background-color: white;
  padding: 14px;
  padding-top: 15px;
  padding-left: 44px;
  padding-right: 44px;
  /* padding-right: 15px;
  border-top-left-radius: 8px;
  border-top-right-radius: 8px; */
}

#low-temp-img{
  position: fixed;
  height: 6%;
  width: 4%;
  top: 25%;
  left: 52%;
  background-color: white;
  padding: 10.5px;
  padding-left: 35px;
  padding-right: 39px;
  /* border-top-left-radius: 8px; */
  /* border-top-right-radius: 8px; */
}

.temp-container {
  background-color: white;
}

.high{
  position: fixed;
  left: 43%;
  top: 30%;
  background-color: white;
  padding: 11px;
  padding-right: 36px;
  padding-left: 32px;
  border-bottom-left-radius: 8px;
  /* border-bottom-right-radius: 8px; */
}

.low{
  position: fixed;
  left: 52%;
  top: 30%;
  background-color: white;
  padding-top: 10px;
  padding-bottom: 12px;
  padding-left: 37px;
  padding-right: 38px;
  /* border-bottom-left-radius: 8px; */
  border-bottom-right-radius: 8px;
}

.current{
  position: fixed;
  left: 43%;
  top: 8%;
  font-size: 64px;
  background-color: white;
  padding: 10px;
  /* border-radius: 8px; */
  border-top-left-radius: 8px;
  border-top-right-radius: 8px;
}
