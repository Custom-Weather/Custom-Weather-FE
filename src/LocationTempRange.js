import React, { Component } from 'react';
import './LocationTempRange.css';

const LocationTempRange = () => {
  return (
    
  )
}
