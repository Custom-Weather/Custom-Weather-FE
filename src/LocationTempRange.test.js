import React from 'react';
import { render, cleanup } from '@testing-library/react';
import LocationTempRange from './LocationTempRange';

afterEach(cleanup)

 it('should take a snapshot', () => {
    const { asFragment } = render(<LocationTempRange />)

    expect(asFragment(<LocationTempRange />)).toMatchSnapshot()
  })
