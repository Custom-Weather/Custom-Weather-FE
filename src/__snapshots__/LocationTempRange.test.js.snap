// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="location-temp-range"
  >
    <h4>
       Today's Temperature Ranges 
    </h4>
    <p>
      Current: 
    </p>
    <p>
      High: 
    </p>
    <p>
      Low: 
    </p>
  </div>
</DocumentFragment>
`;
